openapi/data_repository_service.swagger.yaml

```diff
@@ -46,6 +46,20 @@ paths:
           description: The Data Bundle was found successfully.
           schema:
             $ref: '#/definitions/Bundle'
+        '301':
+          description: >
+            The operation is delayed and will continue asynchronously.
+            The client should follow the redirect after the delay specified in the Retry-After header.
+          headers:
+            Retry-After:
+              description: >
+                Delay in seconds. The client should follow the redirect after waiting for this duration.
```

Comment:
Since Retry-After is also mentioned in the HTTP RFCs, it might make sense to qualify that only the integral relative time in seconds may be returned by the server and that the absolute string date (that's overly complicated to parse) is explicitly disallowed.